src/model/formats/garmin-pilot-reader.ts

```diff
@@ -75,19 +76,18 @@ export class GarminPilotReader {
     checklistGarmin: GarminPilotChecklist,
     checklistItemsEFIS: Map<string, ChecklistItem[]>,
   ): [GarminChecklistGroupKey, Checklist] {
+    const items = [...checklistItemsEFIS]
+      .filter(([itemKey, _checklistItems]) => checklistGarmin.checklistItems.includes(itemKey))
+      .flatMap(([_itemKey, checklistItems]) => checklistItems);
+
     return [
       [checklistGarmin.type, checklistGarmin.subtype],
       {
         title: checklistGarmin.name,
-        items: [
-          ...[...checklistItemsEFIS]
-            .filter(([itemKey, _checklistItems]) => checklistGarmin.checklistItems.includes(itemKey))
-            .flatMap(([_itemKey, checklistItems]) => checklistItems),
-          ChecklistItem.create({
-            type: ChecklistItem_Type.ITEM_PLAINTEXT,
-            prompt: GarminPilotUtils.COMPLETION_ACTION_TO_EFIS.get(checklistGarmin.completionItem),
-          }),
-        ],
+        items,
+        completionAction:
+          GarminPilotUtils.COMPLETION_ACTION_TO_EFIS.get(checklistGarmin.completionItem) ??
```

Comment:
In this case, I guess this might be okay, because I believe that Garmin Pilot always sets the completion action, so this fallback is actually never going to be triggered.

We don't have any guarantees, though, so you might consider the other comment here as well... or not.